uaa/slateCustomizations/source/index.html.md.erb

```diff
@@ -156,9 +156,12 @@ _Response Headers_
 
 # Token
 
-The `/oauth/token` endpoint requires client authentication to be accessed. Client Authentication can be passed as
-as part of the request authorization header, using basic authentication, or as part of the request parameters, using the `client_id` and `client_secret` parameter
-names.
+The `/oauth/token` endpoint requires client authentication to be accessed, except you allow the public usage. For more details about public usage see
+allowpublic flag in client details configuration. For confidential client usages the client authentication can be passed as part of the request
```

Comment:
I find having a `,` after `For confidential client usages` makes it more readable.